apps/omg_watcher/lib/omg_watcher/exit_processor/in_flight_exit_info.ex

```diff
@@ -458,9 +462,24 @@ defmodule OMG.Watcher.ExitProcessor.InFlightExitInfo do
   defp do_is_viable_competitor?(seen_at_pos, oldest_pos, competitor_pos),
     do: is_older?(competitor_pos, seen_at_pos) and is_older?(competitor_pos, oldest_pos)
 
+  # no position is older than any real position
+  defp is_older?(Utxo.position(_, _, _), :no_position), do: true
```

Comment:
is see that this module has private and public functions scattered around. public on top, private below?

Reply:
true, this needs cleanup, but I suggest doing so in a separate PR